src/services/filterDestination.js

```diff
@@ -0,0 +1,26 @@
+export function filterDestination(redirectTo, purposeParams, userID, redirectID){
+    var redirectURL = "";
+    var fullurl = "";
+
+    switch(purposeParams){
+        case 'plio':
+            //this method constructs the URL based on the redirectTo param
+            redirectURL = process.env.VUE_APP_BASE_URL_PLIO;
+            var url = new URL(redirectURL + redirectID); //adds plioID to the base plio link
+            //adds params; api key and student SRN
+            var queryparams = new URLSearchParams({
+                                api_key: process.env.VUE_APP_AF_API_KEY,
+                                unique_id: userID,
+                                });
+            fullurl = url + "?" + queryparams;
+            break;
+
+        case 'liveclass':
+            //this method constructs the URL based on the redirectTo param
+            redirectURL = process.env.VUE_APP_BASE_URL_MEET;
+            fullurl = new URL(redirectURL + redirectID); //adds meetID to the base plio link
+            break;
+        }
```

Comment:
This switch case would not be required if the logic was separated into different functions

Reply:
That was how it was done previously, but @pritamps felt merging the code is better.

Reply:
@pritamps need your help here, what should we do?
If we keep the switch case, if we add more things, we will have to keep adding more `cases`

Reply:
@sudheshna-donthineni I suggested that you change your `if-else` to a `switch-case` to make it clearer. Had nothing to do with methods, really. It was still in the same function. What @deepansh96 is suggesting is that move the logic to different methods for each redirect type. 

I agree with him in principle, but I think the `switch-case` is fine for now. The logic inside each case can be extracted to methods later when the number increases a lot

Reply:
Cool, sounds good to me
Will be super helpful to follow ES6 module system

Then we'll just be able to do this in any file
`import { redirectToPlio } from '@/services/redirection.js'`